main.go

```diff
@@ -83,6 +102,34 @@ func main() {
 	os.Exit(exitCode)
 }
 
+// Check status of SDS secrets from Istio to make sure SDS bootstrap was successful
```

Comment:
Godoc should be complete sentences, needs punctuation.

Reply:
fixed in https://github.com/influxdata/envoy-preflight/pull/4